internal/newsfeed/news_feed_manager.go

```diff
@@ -102,7 +125,7 @@ func (n *NewsFeedManager) StartFetching(ctx context.Context) {
 	ctx, n.cancel = context.WithCancel(ctx)
 
 	// Initial fetch
-	_ = n.fetchRSS()
+	_ = n.fetchRSSAndHandle()
```

Comment:
This initial fetch will happen syncronously 🤔 would be better to do it inside the goroutine below.

There's a trick with cron tasks to trigger instantly (e.g. [here](https://github.com/status-im/status-go/blob/3179532b645549c103266e007694d2c81a7091b4/protocol/messenger_curated_communities.go#L33-L34))
```go
go func() {
// Initial trigger
duration := time.Duration(0)
for {
	select {
	case <-time.After(duration):
        // Set the actual interval
		duration = n.pollingInterval 
```